packages/adp-tooling/templates/project/package.json

```diff
@@ -5,19 +5,22 @@
     "description": "<%= package.description %>",
     "keywords": [
         "ui5",
-        "openui5",
-        "sapui5"
+        "sapui5",
+        "adaptation-project"
     ],
     "dependencies": {},
     "devDependencies": {
-        "@sap-ux/backend-proxy-middleware": "^0.6.68",
-        "@sap-ux/preview-middleware": "^0.5.0",
-        "@sap-ux/ui5-proxy-middleware": "^1.1.34",
+        <%if (locals.options?.fioriTools) {%>"@sap/ux-ui5-tooling": "^1.11.4"<%} else {%>"@sap-ux/backend-proxy-middleware": "^0.7.5",
+        "@sap-ux/preview-middleware": "^0.11.1",
+        "@sap-ux/ui5-proxy-middleware": "^1.3.0",
+        "@sap-ux/deploy-tooling": "^0.11.7"<%}%>,
         "@ui5/cli": "^3.6.0"
     },
     "scripts": {
         "build": "ui5 build --exclude-task generateFlexChangesBundle generateComponentPreload --clean-dest",
-        "start": "ui5 serve --open /test/flp.html#app-preview",
-        "editor": "ui5 serve --open /local/editor.html"
+        "start": "<%= locals.options?.fioriTools ? 'fiori run' : 'ui5 serve' %> --open /test/flp.html#app-preview",
+        "start-editor": "<%= locals.options?.fioriTools ? 'fiori run' : 'ui5 serve' %> --open /test/adaptation-editor.html"<%if (locals.deploy) {%>,
+        "deploy": "ui5 build --config ui5-deploy.yaml --exclude-task <%= locals.options?.fioriTools ? 'deploy-to-abap' : 'abap-deploy-task' %> generateFlexChangesBundle generateComponentPreload --clean-dest && <%= locals.options?.fioriTools ? 'fiori deploy' : 'deploy' %> --config ui5-deploy.yaml",
+        "undeploy": "undeploy --config ui5-deploy.yaml --lrep 'apps/<%= app.reference %>/appVariants/<%= app.id %>/'"<%}%>
```

Comment:
For the Fiori tools use case the commando should be `fiori undeploy`

Reply:
5d3d666